README.md

```diff
@@ -1,2 +1,4 @@
 # dummy-repo
 Dummy repository for testing things
+
+Some dummy content.
```

Comment:
I guess `lorem-ipsum.md` is sufficient.  This dummy content can be simply removed.